router/middleware/logger_test.go

```diff
@@ -217,3 +224,61 @@ func TestMiddleware_Logger_Sanitize(t *testing.T) {
 		}
 	}
 }
+
+func TestMiddleware_Format(t *testing.T) {
+
+	// setup data, fields, and logger
+	formatter := &Formatter{
+		DataKey: "labels.vela",
+	}
+
+	fields := logrus.Fields{
+		"ip":         "123.4.5.6",
+		"method":     http.MethodGet,
+		"path":       "/foobar",
+		"latency":    0,
+		"status":     http.StatusOK,
+		"user-agent": "foobar",
+		"version":    "v1.0.0",
+		"org":        "foo",
+	}
+
+	logger := logrus.NewEntry(logrus.StandardLogger())
+	entry := logger.WithFields(fields)
+
+	got, err := formatter.Format(entry)
+
+	wantLabels := "labels.vela"
+	wantMethod := http.MethodGet
+	wantOrg := "foo"
+
+	// run test
+	gotLabels := string(formatter.DataKey)
+	gotMethod := entry.Data["method"]
+	gotOrg := entry.Data["org"]
```

Comment:
these look like they are checking the "old" field keys? i believe we should be checking the contents of `got` from _after_ the `Format` is called
```
	got, err := formatter.Format(entry)
```

Reply:
addressed